server/routerlicious/patches/@microsoft__api-extractor@7.42.3.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/lib/enhancers/ValidationEnhancer.js b/lib/enhancers/ValidationEnhancer.js
```

Comment:
`server/routerlicious` requires a separate copy of this to work in their docker environment.